controllers/articles.controllers.js

```diff
@@ -20,3 +22,17 @@ exports.getArticleById = (req, res, next) => {
     })
     .catch(next);
 };
+
+exports.patchArticleById = (req, res, next) => {
+  const { article_id } = req.params;
+  const { inc_votes } = req.body;
+
+  checkArticleExists(article_id)
+    .then(() => {
+      return updateArticleById(article_id, inc_votes)
+        .then((article) => {
+        res.status(201).send({ article });
```

Comment:
As per above comment. Remember to alter the test first, see it fail then amend to pass.

Reply:
As above, updated test to 200, updated code to pass failing test :raised_hands: